glean_parser/metrics.py

```diff
@@ -90,6 +91,8 @@ def __init__(
                 getattr(DataSensitivity, x) for x in data_sensitivity
             ]
         self.defined_in = defined_in
+        if telemetry_mirror is not None:
```

Comment:
Should probably just remove the `if` statement so the member is always there, it just might be `None`.  That prevents downstream code from needing to write `if hasattr(metric, "telemetry_mirror")`...

Reply:
As you rightly pointed out, this won't work (for schema validation reasons), so seems fine...